_pages/algorithm-development/languages/python.md

```diff
@@ -366,20 +290,33 @@ from sub_module.special_stuff import special_stuff
 This will work fine for Python 2.  However, for Python 3, you need to use the [dot-prefix notation](https://docs.python.org/3/reference/import.html#submodules) for local files:
 
 ```python
+from Algorithmia import ADK
 import Algorithmia
 import os
 from .secondary_file import auxillary_func, some_other_func
 from .sub_module.special_stuff import special_stuff
 ```
 
+## Algorithm Checklist
+
+Before you are ready to publish your algorithm it's important to go through this [Algorithm Checklist]({{site.baseurl}}/algorithm-development/algorithm-checklist) and check out this blog post for <a href="https://algorithmia.com/blog/advanced-algorithm-design">Advanced Algorithm Development <i class="fa fa-external-link"></i></a>.
+
+Both links will go over important best practices such as how to create a good algorithm description, add links to external documentation and other important information.
```

Comment:
suggest change to:
"These resources provide important information on best practices, including how to write a good algorithm description and how to add links to external documentation."